rotkehlchen/accounting/export/csv.py

```diff
@@ -30,6 +30,7 @@
     'include_gas_costs',
     'account_for_assets_movements',
     'calculate_past_cost_basis',
+    'pnl_order',
```

Comment:
name change, as per my comment on the enum

Reply:
Ok